tests/flow/test_function_calls.py

```
@@ -2438,375 +2438,376 @@ def test87_typeof(self):
+        actual_result = self.graph.query(query)
+        self.env.assertEquals(actual_result.result_set[0], expected_result)
 
-    #    expected_result = ['HELL OW']
-    #    query = """RETURN string.join(['HELL','OW'], ' ')"""
-    #    actual_result = self.graph.query(query)
-    #    self.env.assertEquals(actual_result.result_set[0], expected_result)
+        # 2nd arg should be string
+        try:
+            self.graph.query("RETURN string.join(['HELL','OW'], 2)")
+            self.env.assertTrue(False)
+        except ResponseError as e:
```

Ruff (0.14.13)

`ResponseError` may be undefined, or defined from star imports

(F405)